### DISCUSSION TITLE
Set environment variable for GitHub Actions

### DISCUSSION THREAD

**Discussion by Dwite** [fastlane/fastlane#17878] [Q&A] [answered]
I'm trying to export some data from fastlane actions as environment variable for GitHub Actions to use in next steps as

**Fastfile**:
```
name = android_get_version_name
code = android_get_version_code

ENV[VERSION_NAME] = name
ENV[VERSION_CODE] = code
```

and than use in **GitHub Action** as 

```
env:
  SLACK_MESSAGE: 'Build for ${{ env.VERSION_NAME }}(${{ env.VERSION_CODE }}) is ready for testing.'
```

but looks like it's empty, any ideas?

**Comment by rogerluan** [comment]
Hey @Dwite 👋 

Looks like you have to write to the `GITHUB_ENV` file otherwise that env var will probably only be available within your scope. See https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-commands-for-github-actions#setting-an-environment-variable

**Comment by Dwite** [comment, marked as answer]
Hey @rogerluan, thank you for the link, I was able to solve the question using the following solution in Fastlane script:

```
name = android_get_version_name
code = android_get_version_code

sh("echo VERSION_NAME=#{name} >> $GITHUB_ENV")
sh("echo VERSION_CODE=#{code} >> $GITHUB_ENV")
```

**Reply by rogerluan** [reply]
Awesome! Glad to hear it worked 🚀
